src/app/components/forms/form/terra-form.component.ts

```diff
@@ -202,4 +206,17 @@ export class TerraFormComponent implements ControlValueAccessor, OnChanges, OnIn
     {
         return this._formGroup;
     }
+
+    /**
+     * Clone objects or arrays to prevent instance clash.
+     * @param value to clone if isObject or isArray.
+     */
+    private cloneDefaultValue(value:any):any
+    {
+        if(isObject(value) || isArray(value))
+        {
+            return _.cloneDeep((value));
```

Comment:
Ein Paar Klammern zu viel
```suggestion
            return _.cloneDeep(value);
```